src/parser/ad_parser.js

```diff
@@ -153,7 +153,7 @@ function parseAdElement(adTypeElement, emit) {
         break;
 
       case 'ViewableImpression':
-        ad.viewableImpression = _parseViewableImpression(node);
+        ad.viewableImpression = [_parseViewableImpression(node)];
```

Comment:
As we initialized the ad object with an empty array for viewableImpression we can do the same as l.183